ObjectOrientedProgramming/Memory Ownership/New and delete operators/src/task.cpp

```diff
@@ -0,0 +1,12 @@
+#include <iostream>
+
+void createAndDeleteObject() {
+    // Create an integer object using the new operator.
+    int* p = new int();
```

Comment:
Let's ask the user to create a proper custom object with custom constructor and destructor, in order to demonstrate the difference w.r.t. `malloc`/`free`.